plugins/out_opentelemetry/opentelemetry.c

```diff
@@ -1348,6 +1494,57 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct opentelemetry_context, log_response_payload),
      "Specify if the response paylod should be logged or not"
     },
+    {
+     FLB_CONFIG_MAP_STR, "logs_observed_timestamp_metadata_key", "$ObservedTimestamp",
+     0, FLB_TRUE, offsetof(struct opentelemetry_context, logs_observed_timestamp_metadata_key),
+     "Specify an ObservedTimestamp key"
+    },
+    {
+     FLB_CONFIG_MAP_STR, "logs_timestamp_metadata_key", "$Timestamp",
+     0, FLB_TRUE, offsetof(struct opentelemetry_context, logs_timestamp_metadata_key),
+     "Specify an Timestamp key"
```

Comment:
This and a number of other descriptions should be `a` instead of `an` when the word following it starts with a consonant.

Reply:
Oops. Thank you for pointing it.
I fixed them.
https://github.com/fluent/fluent-bit/pull/8475/commits/7660e3f25d4bf0520878b3c1dc6368b514a90a32